### ISSUE TITLE
Yo trails:policy, policies/index.js issue

### ISSUE THREAD

**Issue by cesar2064** [trailsjs/trails-policy#5]
When I create a policy by using yo trails:policy generator, it adds an export to policies/index.js as follows:

```
'use strict'
exports.Auth = require('./Auth')
```

but when I try to use it, the application logs a a warn:

`warn: trailpack-router: route handler [ Auth.checkAuthentication ] does not correspond to any defined Controller handler`

But it works correctly when I change it manually to :

```
'use strict'
module.exports.Auth = require('./Auth')
```

Environment: node: 6.2.1, npm:3.5.3, windows 10 x64

**Comment by wbprice** [comment]
@cesar2064 are you using Hapi as your webserver?  We recently pushed up a new version of `trailpack-hapi` that removes `module.exports = {}` from the default `api/policies/index.js`, fixing this issue. 

The preferred workaround in the meantime is to delete the line containing `module.exports = {}` in `api/policies/index.`  That way, any other policies that you create will work without modifying their require statements.

**Comment by cesar2064** [comment]
@wbprice no I am not. I am using express.

**Comment by wbprice** [comment]
What version of `trails-express` is installed? Latest is 1.0.9. I'll see if I can replicate with a fresh Express install later.

**Comment by wbprice** [comment]
@cesar2064 The fix for this issue was published on Jun 12 in `trails-express@1.0.9`, so fresh installs of Trails won't run into the issue raised above.  In your case, removing `module.exports = {}` from `api/policies/index.js` will be OK.

**Comment by cesar2064** [comment]
Yes @wbprice It fixs the issues. Thanks for your help.
